modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc

```diff
@@ -1638,27 +1638,144 @@ Returns `NULL` if either or both expressions are `NULL`.+
+
+* `fmt` - An optional string parameter representing a date format to output the result as.
+|Returns a date string representing the date in the specified timezone in the specified format.
+|If `fmt` is not specified, the output default to the combined full date and time.
+|`DATE_DIFF_STR(date1, date2, part)`
+|Finds the elapsed time between two date strings.
+This is measured from `date2` to `date1`.
+a|
+
+* `date1` - A valid date string. 
+This is converted to UTC.
+
+* `date2` - A valid date string. 
+This is converted to UTC.
+
+* `part` - A string representing the component of the date to increment.
+|Returns an integer representing the elapsed time (based on the specified `part`) between both dates.
+a|
+
+* If `date1` is greater than `date2` then the value returned will be positive, otherwise the returned value is negative.
+
+* If `date1` or `date2` are not valid string formats, `null` is returned.
+
+* If an invalid `part` is passed to the function, `null` is returned.
+|`DATE_DIFF_MILLIS(date1, date2, part)`
+|Finds the elapsed time between two Epoch/UNIX timestamps. This elapsed time is measured from `date2` to  `date1`. 
+a|
+
+* `date1` - An integer representing an Epoch/UNIX timestamp in milliseconds.
+
+
+* `date2` - An integer representing an Epoch/UNIX timestamp in milliseconds.
 
+* `part` - A string representing the component of the date to increment.
+|Returns an integer representing the elapsed time (based on the specified `part`) between both dates.
+a|
 
+* If `date1` is greater than `date2` then the value returned will be positive, otherwise the returned value is negative.
+
+* If `date1` or `date2` are not integers, `null` is returned.
+
+* If an invalid `part` is passed to the function, `null` is returned.
+|`DATE_ADD_STR(date1, n, part)`
+|Performs date arithmetic on a date string. This calculation is specified by the arguments `n` and `part`. For example a value of `3` for `n` and a value of `day` for `part` would add 3 days to the date specified by `date1`.
+a|
+
+* `date1` - A valid date string. 
+This is converted to UTC.
+
+
+* `n` - An integer or expression that evaluates to an integer.
+A positive value will increment the date component whereas a negative value will decrement the date component.
+
+* `part` - A string representing the component of the date to increment.
+|Returns an integer representing the calculation result as an Epoch/UNIX timestamp in milliseconds.
+a|
+
+* If `date1` is not a valid string format, `null is returned`.
+
+* If a non-integer is passed to the argument `n`, `null` is returned.
+
+* If an invalid `part` is passed to the function, `null` is returned.
+
+|`DATE_ADD_MILLIS(date1, n, part)`
+|Performs date arithmetic on a particular component of an Epoch/UNIX timestamp value. This calculation is specified by the arguments `n` and `part`. 
+For example a value of `3` for `n` and a value of `day` for `part` would add 3 days to the date specified by `date1`.
```

Comment:
This would be simpler with a code example:

"For example `DATE_ADD_MILLIS(10000000000, 3, 'day')` would add 3 days to the date specified."

(Double check the syntax, e.g. I didn't check if that should be `'day'` or `day` or `"day"`)